packages/axelar-wasm-std/src/address.rs

```diff
@@ -43,6 +44,19 @@ pub fn validate_address(address: &str, format: &AddressFormat) -> Result<(), Err
             ScAddress::from_str(address)
                 .change_context(Error::InvalidAddress(address.to_string()))?;
         }
+        AddressFormat::Base58Solana => {
+            const SOLANA_PUBKEY_LEN: usize = 32;
+            const MAX_BASE58_LEN: usize = 44;
+            if address.len() > MAX_BASE58_LEN {
+                bail!(Error::InvalidAddress(address.to_string()));
+            }
+            let pubkey_vec = bs58::decode(address)
```

Comment:
Is this implementation malleable? e.g. can you have two strings decode to the same pubkey? Or is it strict?

Reply:
No, it's strict in that sense (bijective).